demo/billing/src/error.rs

```diff
@@ -19,9 +19,6 @@ pub enum Error {
     #[error("Unexpected error running postgres command")]
     PgError(#[from] tokio_postgres::Error),
 
-    #[error("Interacting with kafka: {0}")]
```

Comment:
probably the best thing to do long term (no need to do so now) is to use anyhow in this demo